### ISSUE TITLE
This plugin generated a warning from the characteristic 'Target Temperature'

### ISSUE THREAD

**Issue by wawoodwa** [lukasroegner/homebridge-dyson-pure-cool#165]
Getting this error in the Homebridge log:

```
[3/1/2021,`` 10:11:01 AM] [homebridge-dyson-pure-cool] This plugin generated a warning from the characteristic 'Target Temperature': characteristic was supplied illegal value: number 37.19999999999999 exceeded maximum of 37. See https://git.io/JtMGR for more info.
```
Error occurs every minute.

**Comment by wawoodwa** [comment]
Anything I can do to help track this down?

**Comment by wawoodwa** [comment]
Closing issue.  Updated version of HomeBridge...error has ceased.

**Comment by wawoodwa** [comment]
Error has come back.  However, I think I know the issue.  The device was set to 99F which is higher than the 37C the plugin accepts.  Perhaps increase the limit to 38C?

**Comment by EdwinJS** [comment]
Thanks for doing this Wawoodwa, gonna bump this so there's a pull.

**Comment by lukasroegner** [comment]
Fixed in 1.8.8, thanks to @wawoodwa
